bpf/bpf_lxc.c

```diff
@@ -685,6 +684,18 @@ int tail_handle_ipv6(struct __ctx_buff *ctx)
 			   CILIUM_CALL_IPV6_CT_EGRESS, tail_ipv6_ct_egress);
 	return ret;
 }
+
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_FROM_LXC)
+int tail_handle_ipv6(struct __ctx_buff *ctx)
+{
+	int ret = __tail_handle_ipv6(ctx);
+
+	if (IS_ERR(ret)) {
+		return send_drop_notify(ctx, SECLABEL, 0, 0,
+			ret, CTX_ACT_DROP, METRIC_EGRESS);
+	}
```

Comment:
Please follow the established style - indentation of the function arguments, no curly braces around single statements. See the other `send_drop_notify()` calls in the same file.

Reply:
Done.